app/templates/home.html

```diff
@@ -2,31 +2,21 @@
 
 {% block content %}
 
-
   <div class="container mt-4">
-  </div>
-
-
-  <!-- I chose to add the quote day to the home page until the relavent calendar view will be developed.  -->
-  <div>
     {% if quote %}
       {% if not quote.author %}
         <p><i>"{{ quote.text }}"</i></p>
       {% else %}
-        <p><i>"{{ quote.text }}"</i> <span style="font-size: small;">&nbsp; \ {{ quote.author }}</span></p>
-      {% endif %}
-    {% endif %}
-  </div>
-
-<!-- I chose to add the quote day to the home page until the relavent calendar view will be developed.  -->
-<div>
-    {% if quote %}
-        {% if not quote.author%}
-            <p><i>"{{ quote.text }}"</i></p>
+      <p><i id="quote">"{{ quote.text }}"</i> <span id="author" style="font-size: small;">&nbsp; \ {{ quote.author }}
+        {% if quote.is_favorite %} 
+          <img src="../media/full_heart.png" class="heart">
```

Comment:
url_for

Reply:
Can we delete this file? I think everything in here moved to index.html and we don't need it anymore.